### PR TITLE
Upgrade Everrest to 1.13.5 version

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Upgrade  Everrest to newer version
 
List of related CQs:

- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13883 Everrest 1.3.5
- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13884 commons-fileupload 1.3.3

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/5694

### CODE REVIEW

**Review by riuvshin — APPROVED** [review]

**Review by skabashnyuk — COMMENTED** [review]
I'm ok to merge after CQ approval

**Review by skabashnyuk — APPROVED** [review]

**Pull request merged**
